blog/2024-12-12-map/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Making sense of Kubernetes with Map view"
+description: "Introducing Map View feature to Headlamp"
+image: ./splash.jpg
+authors: ["olek"]
+---
+
+import mapDemo from "./map-demo.mp4";
+import identifyProblems from "./identify-problems.mp4";
+
+In version 0.26, Headlamp introduced a new feature: the Map View. It provides a graphical way to see how components like pods, services, and deployments interact in your Kubernetes cluster. This makes it easier to understand dependencies, troubleshoot issues, and optimize your cluster setup.
+
+<!--truncate-->
+
+## About
+
+<p>
+  <video controls width="100%">
+    <source src={mapDemo} type="video/mp4" />
+  </video>
+</p>
+
+Instead of working with tables or YAML files, you can now visually explore the connections between resources. For example, you can see which pods are linked to which services or how deployments relate to replica sets. This makes it much simpler to understand your cluster’s structure.
+
+When troubleshooting, the Map View is especially useful. If a pod fails, you can find which services or deployments depend on it, helping you identify the cause of the issue.
+
+<p>
+  <video controls width="100%">
+    <source src={identifyProblems} type="video/mp4" />
+  </video>
+</p>
+
+We’re also working on providing APIs for plugins, allowing you to extend the Map View with extra details.
```

Comment:
This seems a bit dangling/unrelated. I think it's a good thing to add to the conclusion as part of mentioning the future.
I had actually assumed that you were showing examples of the plugin API results because of having the examples section just below this statement.